package.json

```
@@ -47,6 +47,7 @@
   "license": "MIT",
   "dependencies": {
     "@types/debug": "^4.1.7",
+    "@types/retry-as-promised": "2.3.4",
```

We don't pin our production dependencies so tools like `npm audit fix` continue working :)

```suggestion
    "@types/retry-as-promised": "^2.3.4",
```